pokerbot.js

```diff
@@ -23,19 +24,20 @@ pokerbot.root = function (req, res, next) {
   const channelId = req.body.channel_id
   console.log('Channel Id  : ' + channelId)
   console.log('Token Id  : ' + token)
+
   // Bad command syntex.
-  if ((option !== 'start' && option !== 'stop') || jiraId === undefined) {
+  if ((option !== 'start' && option !== 'stop' && option !== 'status') || jiraId === undefined) {
```

Comment:
This might be cleaner if written this way:
```
const validCommand = _.includes(['start', 'stop', 'status'], option)
if (validCommand || jiraId === undefined) {
...
```